src/main/resources/io/jenkins/plugins/analysis/warnings/axivion/local-config.jelly

```diff
@@ -0,0 +1,18 @@
+<?jelly escape-by-default='true'?>
```

Comment:
This file needs to be renamed to config.jelly! local-config.jelly is only for subtypes of `ReportParsingTool`.